rero_ils/modules/documents/templates/rero_ils/_documents_get.html

```diff
@@ -135,7 +135,7 @@
         <!-- display items -->
         {% if number_items > 0 %}
         {% for item in items %}
-        <div class="row my-2">
+        <div id="{{item.barcode}}-detail" class="row my-2">
```

Comment:
best practice : add spaces between curly braces and variable

Reply:
Done